src/components/design/nav/index.js

```diff
@@ -59,7 +59,12 @@ function Nav(props) {
             Array.isArray(menu) &&
             menu.map((item, index) => (
               <li key={index}>
-                <a href={item.url} target="_blank" rel="noopener noreferrer">
+                <a
+                  href={item.url}
+                  target={item.value === 'Community' || item.value === 'Docs' ? '_blank' : null}
+                  rel="noopener noreferrer"
+                  onClick={drawerOnClick}
+                >
```

Comment:
Nice catch! I suggest using the same approach used on the main navbar (:35) to reduce the `<a>` attributes to the essential ones when it is an internal link and external. What do you think?

```suggestion
                {item.external ? (
                  <a href={item.url} target="_blank" rel="noopener noreferrer" onClick={drawerOnClick}>
                    {item.value}
                  </a>
                ) : (
                  <a href={item.url} onClick={drawerOnClick}>
```

Reply:
hey just seen your comment :) I'll have a look at it this month